### PR TITLE
PROGRAMMES-5735 - Fixing broadcast details spacing issue

### PR DESCRIPTION
**Pull request by felipeparaujo**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/DsAmen/Organism/Programme/_collapsed-broadcast.scss [hunk 1]
```diff
@@ -3,8 +3,8 @@
  */
 
 .broadcast-details__service {
-    span[property='name'] {
-        margin-right: $gel-unit-small / 2;
+    span ~ span {
```
Comment (votemike) [inline src/DsAmen/Organism/Programme/_collapsed-broadcast.scss]: What is this? Can you not use class names instead?

Comment (felipeparaujo) [reply src/DsAmen/Organism/Programme/_collapsed-broadcast.scss]: This means that every span preceded by a span gets that property.

Comment (felipeparaujo) [reply src/DsAmen/Organism/Programme/_collapsed-broadcast.scss]: I could use classes, but it would just be the same

Comment (votemike) [reply src/DsAmen/Organism/Programme/_collapsed-broadcast.scss]: span ~ span is gross and confusing. What if it is `<span><span><span></span></span></span>`?

Comment (felipeparaujo) [reply src/DsAmen/Organism/Programme/_collapsed-broadcast.scss]: None of them gets the styling. This makes it easier to implement stuff like "add spacing to all children except for the first one". If I remove this, the template code will look uglier.

**Pull request merged**